tests/Unit/Commands/RunBackupCommandTest.php

```
@@ -121,4 +122,19 @@ public function test_runs_enabled_backup_even_after_a_failed_run(): void
         $this->artisan('backups:run')
             ->expectsOutput('1 backups started');
     }
+
+    public function test_does_not_run_backups_whose_server_is_missing(): void
+    {
+        SSH::fake();
+        Bus::fake();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Bus' in method 'test_does_not_run_backups_whose_server_is_missing'. (undefined)

(StaticAccess)